torch_ttnn/passes/lowering/add_data_move_pass.py

```diff
@@ -124,15 +124,22 @@ def is_function_call(node) -> bool:
     ttnn.split,
     ttnn.slice,
     ttnn.to_layout,
+    ttnn.sharded_to_interleaved,
 ]
 
-TTNN_TARGET_WRAPPERS = [target_wrappers.clone, target_wrappers.repeat]
+TTNN_TARGET_WRAPPERS = [
+    target_wrappers.clone,
+    target_wrappers.repeat,
+    target_wrappers.pack_to_tuple,
+]
 
 TTNN_NORM_OPS = [
     ttnn.group_norm,
     ttnn.layer_norm,
 ]
 
+TTNN_POOL_OPS = [ttnn.max_pool2d]
```

Comment:
there is also `avg_pool2d`

Reply:
we can probably move it in here from line 179

Reply:
Done